.prettierignore

```diff
@@ -3,27 +3,25 @@
 .changeset
 .contentlayer
 .DS_Store
-.husky
 .env.local
 .eslintignore
+.eslintrc
 .gitignore
 .husky
 .next
-.prettierignore
-.next
 .node-version
 .npmrc
 .nvmrc
+.prettierignore
 
 coverage
 dist
-docs/.cache
-docs/public/render-page.js.map
 LICENSE
 storybook-static
 pnpm-lock.yaml
+public
 
-*.png
+favicon.png
```

Comment:
Why this change? All png's can be ignored or?

Reply:
sadly no :D

https://github.com/marigold-ui/marigold/pull/4055/files#diff-0a384ae6c978e8d201e4f8a3b211a6839a6669586d575a9b14b0065ab099f49b